src/run_cmd.c

```diff
@@ -77,13 +85,13 @@ static int start_proc(t_mshell *mshell, char *filename, t_cmd cmd)
 	}
 	else
 	{
-		if (child_pid)
-			waitpid(child_pid, &exit_status, 0);
 		argvp = string_to_array(cmd.argc, cmd.argv);
 		if (!argvp)
 			error(E_ALLOC "'start_proc'", mshell);
 		envp = env_to_envp(mshell);
 		ret = execve(filename, argvp, envp);
+		free(filename);
+		error("Unknown command" , mshell);
```

Comment:
Hier ff ms_perror van maken enzo

Reply:
Die hoort uit de shell te exitten though. Kan er msperror van maken maar dan moet hij daarna alsnog erroren. Is voor het vermijden van die 2 processes die runnen.

Reply:
Ohja tuurlijk :)

Reply:
Ohja tuurlijk :)